tremor-pipeline/src/query.rs

```diff
@@ -660,6 +660,7 @@ impl Query {
                 insights: Vec::new(),
                 dot: format!("{}", dot),
                 metrics_channel: METRICS_CHANNEL.tx(),
+                logging_channel: LOGGING_CHANNEL.tx(),
```

Comment:
It might be worth considering a subscriber that subscribes directly to the logging framework instead of weaving the sender throughout the entire runtime - that could save a lot of effort and allow the `error!`, `info!`, `warn!` etc macro to be used without having to re-write every log message.

Reply:
marking this as resolve as there is now a appended :)